usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -9444,6 +9467,19 @@ CK_RV ep11tok_wrap_key(STDLL_TokData_t * tokdata, SESSION * session,
         return rc;
     }
 
+    rc = template_attribute_get_ulong(wrap_key_obj->template, CKA_KEY_TYPE, &ktype);
+    if (rc != CKR_OK) {
+        TRACE_ERROR("Invalid key type attribute\n");
+        return rc;
```

Comment:
this must be `goto done;` instead of return, otherwise we leak the wrap_key_obj.